app/vep/models/submission_form.py

```diff
@@ -11,17 +13,21 @@ class Dropdown(BaseModel):
     options: list[DropdownOption]
     default_value: str
 
+
 class Checkbox(BaseModel):
     label: str
     description: str | None = None
     type: str = "boolean"
     default_value: bool = True
 
+
 class FormConfig(BaseModel):
     transcript_set: Dropdown
     symbol: Checkbox = Checkbox(label="Gene symbol")
     biotype: Checkbox = Checkbox(label="Transcript biotype")
 
+
 class GenomeAnnotationProvider(BaseModel):
-  annotation_provider_name: str
-  annotation_version: str
+    annotation_provider_name: str
+    annotation_version: str
+    last_geneset_update: str
```

Comment:
what is this update for?

Reply:
I noticed that the code (see [here](https://github.com/Ensembl/ensembl-web-tools-api/blob/14cf9ea0c002388cfa2b5b92c4d4b13b0b903d72/app/vep/utils/web_metadata.py#L31) and [here](https://github.com/Ensembl/ensembl-web-tools-api/blob/14cf9ea0c002388cfa2b5b92c4d4b13b0b903d72/app/vep/vep_resources.py#L231)) expects `GenomeAnnotationProvider` instance to include `last_geneset_update` but it's not reflected in the type class so I updated it accordingly. Also it seems the attributes names are missing `.genebuild` prefix.